WebRoot/js/zimbraMail/share/model/ZmFolder.js

```diff
@@ -140,6 +141,7 @@ ZmFolder.QUERY_NAME[ZmFolder.ID_AUTO_ADDED]		= "Emailed Contacts";
 ZmFolder.QUERY_NAME[ZmOrganizer.ID_BRIEFCASE]	= "briefcase";
 ZmFolder.QUERY_NAME[ZmFolder.ID_CHATS]			= "chats";
 ZmFolder.QUERY_NAME[ZmFolder.ID_SYNC_FAILURES]	= "Error Reports";
+ZmFolder.QUERY_NAME[ZmFolder.ID_FILE_SHARED_WITH_ME]	= "Files shared with me";
```

Comment:
please check if folder name gets translated or not

Reply:
Folder name translation is handling by https://github.com/Zimbra/zm-web-client/pull/709/files#diff-063af5f64bad5434cc8f4edf1ee6b098b1984d0a3049fdad361dab0de0015c7dR115 this code.